README.md

```diff
@@ -30,7 +29,7 @@ data Options
   }
   deriving (Show, GHC.Generics.Generic)
 
-instance Generics.SOP.Generic Options
+instance System.Console.GetOpt.Generics.Generic Options
```

Comment:
Yes, that is what I assumed + this is why I left it like that.  In the other examples and specs it's like you how you say.  But for the docs it makes sense to tell the user that there are two classes with the same name and where they come from.